### PR TITLE
Update jp.m3u

### PR DESCRIPTION
**Pull request by BellezaEmporium**

### CODE REVIEW

**Review by Dum4G — DISMISSED** [review]

**Review by Dum4G — APPROVED** [review]

File: streams/jp.m3u [hunk 1]
```diff
@@ -41,6 +41,10 @@ https://n24-cdn-live.ntv.co.jp/ch01/index.m3u8
 https://n24-cdn-live.ntv.co.jp/ch02/index.m3u8
 #EXTINF:-1 tvg-id="ShopChannel.jp",Shop Channel (1080p) [Not 24/7]
 https://stream3.shopch.jp/HLS/master.m3u8
+#EXTINF:-1 tvg-id="JORXDTV.jp",TBS (540p)
+https://tbs4.mov3.co/hls/tbs.m3u8
+#EXTINF:-1 tvg-id="",TBS (540p)
+https://tbs5.mov3.co/hls/tbs.m3u8
```
Comment (Dum4G) [inline streams/jp.m3u:47]: image
not so fast ahaha

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/jp.m3u (hunk 1 above)
Comment (BellezaEmporium) [reply streams/jp.m3u:47]: well lol, that host was known for restreaming many channels... now we know that's pages hahaha

Comment (BellezaEmporium) [reply streams/jp.m3u:47]: It came back it seems.

Comment (BellezaEmporium) [reply streams/jp.m3u:47]: Compared to an original source... that's day and night.
vlcsnap-2024-11-20-12h17m42s944

**Review by Dum4G — APPROVED** [review]

**Review by PopeyeTheSai10r — APPROVED** [review]

**Comment by Hatemmohamme** [comment]
peace for you 
could you help me to send  intact m3u file to me then i will best thank for you and your family

**Comment by Dum4G** [comment]
> peace for you could you help me to send intact m3u file to me then i will best thank for you and your family

https://iptv-org.github.io/iptv/index.m3u

**Pull request merged**